src/test/java/com/es/phoneshop/model/product/ArrayListProductDaoTest.java

```diff
@@ -3,7 +3,10 @@
 import org.junit.Before;
 import org.junit.Test;
 
-import static org.junit.Assert.assertTrue;
+import java.math.BigDecimal;
+import java.util.Currency;
+
+import static org.junit.Assert.*;
 
 public class ArrayListProductDaoTest
```

Comment:
Please ensure that you have separate test cases for methods of this class.
Define concrete class that you're going to test (i.e. ProductDao -> ArrayProductListDao)
Also consider following my other recommendations from our call, when implementing tests (setup() usage, notes about 'expected' exceptions, and etc)